src/main/java/io/vertx/kafka/admin/impl/AdminUtilsImpl.java

```diff
@@ -0,0 +1,167 @@
+package io.vertx.kafka.admin.impl;
+
+import org.I0Itec.zkclient.ZkClient;
+import org.I0Itec.zkclient.ZkConnection;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+import io.vertx.codegen.annotations.VertxGen;
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import io.vertx.kafka.admin.AdminUtils;
+import kafka.utils.ZKStringSerializer$;
+import kafka.utils.ZkUtils;
+
+@VertxGen
+public class AdminUtilsImpl implements AdminUtils {
+  private Vertx vertx;
+  private final String zookeeper;
+  private final boolean isSecure;
+  private int connectionTimeoutMs;
+  private ZkClient zkClient;
+  private boolean autoClose = false;
+
+  public AdminUtilsImpl(Vertx vertx, String zookeeperHosts, int connectionTimeoutMs, boolean isSecure, boolean autoClose) {
+    this.vertx = vertx;
+    this.connectionTimeoutMs = connectionTimeoutMs;
+    this.zookeeper = zookeeperHosts;
+    this.isSecure = isSecure;
+    this.autoClose = autoClose;
+  }
+
+  public AdminUtilsImpl(Vertx vertx, String zookeeperHosts) {
+   this(vertx, zookeeperHosts, 8000, false, false);
+  }
+
+  public AdminUtilsImpl(Vertx vertx, String zookeeperHosts, boolean autoClose) {
+    this(vertx, zookeeperHosts, 8000, false, autoClose);
+  }
+
+  @Override
+  public void createTopic(String topicName, int partitionCount, int replicationFactor,
+                          Handler<AsyncResult<Void>> completionHandler) {
+    createTopic(topicName, partitionCount, replicationFactor, new HashMap<>(), completionHandler);
+  }
+
+  @Override
+  public void createTopic(String topicName, int partitionCount, int replicationFactor,
+                          Map<String, String> topicConfig,
+                          Handler<AsyncResult<Void>> completionHandler) {
+    Properties topicConfigProperties = new Properties();
+    topicConfigProperties.putAll(topicConfig);
+
+    vertx.executeBlocking(future -> {
+      try {
+        kafka.admin.AdminUtils.createTopic(initZkClientAndGetZkUtils(), topicName, partitionCount, replicationFactor, topicConfigProperties,
+          kafka.admin.AdminUtils.createTopic$default$6());
```

Comment:
Since 0.10.1.0 (AFAIK), AdminTools requires you to set a "RackAwareMode" -- this is set here.  
On the command line, you do not have to pass it. Even in Kafka's scala API it is optional -- but Java doesn't know about Scala's optional stuff, so you need to pass it in